src/runtime/js/ts.ui/tables/tables-api@tradeshift.com/models/cols/ts.ui.TableColModel.js

```diff
@@ -148,6 +148,13 @@ ts.ui.TableColModel = (function using(chained) {
 			this.visible = false;
 		}),
 
+		/**
+		 * Cells in this column are sortable? Please note that
+		 * this is could be sort only the talbe is sortable!
```

Comment:
how about 'this works only on the sortable tables!' ? 🤔

Reply:
👍